### PR TITLE
data migration: continuous data validation goes GA

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

DM continuous data validation goes GA.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by D3Hunter — APPROVED** [review]

File: dm/dm-continuous-data-validation.md [hunk 1]
```diff
@@ -5,10 +5,6 @@ summary: 了解增量数据校验的原理，以及如何使用增量数据校
 
 # DM 增量数据校验
 
-> **警告：**
```
Comment (shichun-0415) [inline dm/dm-continuous-data-validation.md:8]: 确认是否移除 [实验特性清单](https://docs.pingcap.com/zh/tidb/dev/experimental-features#%E6%95%B0%E6%8D%AE%E8%BF%81%E7%A7%BB) 里这个 feature。

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

File: dm/dm-continuous-data-validation.md (hunk 1 above)
Comment (hfxsd) [reply dm/dm-continuous-data-validation.md:8]: done

**Pull request merged**